devtools-ci-index.yaml

```
@@ -3777,6 +3874,22 @@
             ee_test_start_time: '20 6,18 * * *'
             timeout: 60m
             zabbix_enabled: false
+        - 'devtools-prebooster-test-{test_url}-{booster_name}':
+            test_url: openshift.io
+            prebooster_test_start_time: '5 0-23/4 * * *'
+            os_creds_username: 'username_1'
+            os_creds_password: 'password_1'
+            project_name: osio-ci-boost1-preview-stage
```

```suggestion
            project_name: osio-ci-preboost1-stage
```
You are taking a username and password from the vault `devtools-osio-ci/osio-ci-prebooster-creds` where the user is `osio-ci-preboost1` so the project name will be as suggested.

---

Ah - good catch - thanks!